sdk/core/core/src/az_json_string_private.h

```diff
@@ -36,6 +37,14 @@ AZ_NODISCARD az_result _az_span_reader_read_json_pointer_token(az_span* self, az
  */
 AZ_NODISCARD az_result _az_span_reader_read_json_pointer_token_char(az_span* self, uint32_t* out);
 
+AZ_NODISCARD AZ_INLINE az_json_token az_json_token_span(az_span span)
```

Comment:
Since this is private, start with `_`.
```suggestion
AZ_NODISCARD AZ_INLINE az_json_token _az_json_token_span(az_span span)
```

Also, where is this method called/used?

Reply:
@vhvb1989 should look at this

Reply:
to be removed. No need to change here